public/scripts/ci.js

```diff
@@ -0,0 +1,61 @@
+function search () {
+  $("#search_result").empty();
+  if ($("#search_bar").val() === "") {
+    $("#search_result").append('<p class="text-center">Please enter the repository name<p>');
+  } else {
+    $("#search_result").append('<p class="text-center">Fetching data<p>');
+    $.get(`https://api.github.com/search/repositories?q=user:${username}+fork:true+${$("#search_bar").val()}`,
+    (result) => {
+      let addDisable = function () {
```

Comment:
I didn't find the usage of this function outside the scope. That's why i scoped inside